<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterEditDlg</class>
 <widget class="QDialog" name="CharacterEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Class</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fClass"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Race</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fRace"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fLevel"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Damage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fDamage"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Armor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fArmor"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Max HP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fMaxHP"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>HP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fHP"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>HP Bonus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fHPBonus"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Morale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fMorale"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Experience</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fExperience"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fSpeed"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>STR</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="fSTR"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>INT</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="fINT"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>WIS</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="fWIS"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>DEX</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="fDEX"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>CON</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="fCON"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>CHA</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="fCHA"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Spells</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="fSpells"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CharacterEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CharacterEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
